pkg/applylib/applyset/applyset.go

```diff
@@ -103,11 +106,11 @@ func (a *ApplySet) ApplyOnce(ctx context.Context) (*ApplyResults, error) {
 
 	for i := range trackers.items {
 		tracker := &trackers.items[i]
-		obj := tracker.desired
+		expectedObject := tracker.desired
```

Comment:
Nit: should probably match the name of the tracker field.  So I would argue either to call this desiredObj or to rename track.desired to tracker.expected.

(Personally I think desired better captures the idea that this is what the user wants, but open either way ... we used `e` for expected in all our kOps tasks)